ports/planetmerge/README.md

```diff
@@ -0,0 +1,12 @@
+# Planet Merge
+
+Source: https://github.com/psiroki/dinnye/
+
+## Controls
+
+| Button | Action |
+|--|--| 
+|DPAD|Move drop position or menu selection|
+|A/B/X/Y|Drop planet|
+|START/MENU|Enter menu|
+|START+SELECT|Exit immediately|
```

Comment:
```suggestion
|START+SELECT|Exit immediately|


##  Compile Instructions
git clone https://github.com/psiroki/dinnye.git
mkdir build && cd build
cmake ..
make
```

Reply:
It needs `-DPORTMASTER=ON` as well, I've added the correct version below.